polaris-react/src/components/ShadowBevel/ShadowBevel.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import type {ComponentMeta} from '@storybook/react';
+import {Box, VerticalStack, InlineCode} from '@shopify/polaris';
+
+import {ShadowBevel} from './ShadowBevel';
+
+export default {
+  component: ShadowBevel,
+} as ComponentMeta<typeof ShadowBevel>;
+
+export function Default() {
+  return (
+    <VerticalStack gap="5">
```

Comment:
Perhaps consider adding some stories with colored backgrounds to show the bevel on something other than a plain white background

Reply:
Good call! Done https://github.com/Shopify/polaris/pull/9638/commits/289528f65d71c244b15f2becc18f8a5c2578d4ec

![image](https://github.com/Shopify/polaris/assets/32409546/faa40ccd-8514-4a0e-83c4-268f5baf88f9)